Шлюзовое

Artemonim opened the pull request:
- [x] ~~#1207~~ Discord проголосовал против
- [x] ~~#2301~~ не получилось воспроизвести
- [x] close #2603
- [ ] close #3206
- [ ] close #3935
- [x] close #4329
- [x] ~~close #4374~~ баг не подтверждён
- [ ] close #4386
- [x] close #4521 TODO: проверить болты и braces
- [ ] close #5472
- [ ] close #5793

```yml
🆑
bugfix: Дюп кабеля при разборке шлюзов
tweak: Неразумные существа не могут открывать двери
rscadd: шлюзы можно прожечь термитом
/🆑
```
<details>
<summary>Чейнджлог</summary>

- bugfix
- balance
- tweak
- soundadd
- sounddel
- rscadd
- rscdel
- imageadd
- imagedel
- maptweak
- spellcheck
- experiment
- admin
</details>

- [ ] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Artemonim committed:
Задержка при заваривании шлюзов, рефакторинг

Zert0X commented:
![image](https://user-images.githubusercontent.com/42303535/125256254-f71a4380-e304-11eb-88c3-f20ba0988e5d.png)

![image](https://user-images.githubusercontent.com/42303535/125256208-e8cc2780-e304-11eb-9d8c-3fa1940f6315.png)

kcalbCube commented:
удали оффтоп собака.

malboec commented:
#5361 частично пофиксил вроде бы

kreeperHLC commented:
![image](https://user-images.githubusercontent.com/14943437/125310818-978b5a80-e33b-11eb-8b5c-22a8ae286ea5.png)
> Требуется проработка

А геймдизы знают?

Artemonim committed:
fix: дюп кабеля при разборке

Большая часть инструментов теперь могут кликнуть на спрайт лишь один раз за операцию

Artemonim committed:
Откат задержки заваривания

[Откат по результатам голосования в Discord](https://discord.com/channels/414832443384659968/606411392811139072/865238614211428412)

Artemonim commented:
@kreeperHLC убирай "рано"

Artemonim committed:
Squashed commit of the following:

commit deb062bde0349c34fa0c53ccfea0c50c510146e4
Author: Artemonim <Artemonim@yandex.ru>
Date:   Mon Jul 5 14:45:05 2021 +0300

    Squashed commit of the following:

    commit 866093683e09e56a5097019220d0f0ea0b92abad
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Mon Jul 5 00:04:45 2021 +0300

        Changelog 04/07/21 (#5920)

    commit 69d6a13922e7b15344d7061404525d2c36dd2701
    Author: Doster <63081538+Doster-d@users.noreply.github.com>
    Date:   Sun Jul 4 17:56:02 2021 +0300

        fix(build): items can no longer be thrown out of vents, banned players see some info

        PR #5903
        - Из труб больше нельзя бросать предметы.
        - Теперь при заходе с баном, помимо причины, выдается ссылка на то, куда писать, чтобы подать апелляцию.

    commit 1db4183f040b1f7aee5097bda2e48eb3f409e859
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sun Jul 4 20:54:19 2021 +0600

        fix(tape recorder): fixes broken icon_states

        PR #5906
        - Диктофоны больше не невидимые.

    commit 807fb25ff3bc37e673b8d9854ee285adfb7220f5
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sun Jul 4 02:06:26 2021 +0600

        fix(floor): fixes floor tiles dupe

        PR #5900
        - Исправлена возможность дюпать плитку, отдирая её с тайла, под которым есть стена.

    commit a3ef0d3886c45c1959698b62b41cace7d879d788
    Author: Epicus <epicus7@gmail.com>
    Date:   Sat Jul 3 19:03:59 2021 +0300

        tweak(storyteller): adds more roles to storyteller

        PR #5500
        - Добавляем больше ролей в сторителлера. Поправил рантайм при переключении вкладок панели управления сторителлером.
        - При спавне борера, мага и ниндзи сторителлер опрашивает гостов и игроков в лобби с соответствующим префенсом, хотят ли они быть ролью. Если игрок отклоняет или не отвечает в течение 10 секунд, то опрашивается следующий игрок.
        - Нужно еще допилить борерам перенос тушки КУДА-ТО - пока не придумал где их спавнить можно.

    commit 5fca2be531faf1049396ab76ae118dcbb2eed5df
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Sat Jul 3 16:59:55 2021 +0300

        tweak(fonts): removes Shage

        PR #5871
        ...whatever it was

    commit c3bc73cf9944b1761e0c3029a3a45a3edfbfdb75
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sat Jul 3 19:57:45 2021 +0600

        tweak(vending machines): makes UI buttons thinner

        PR #5867

    commit 30deb75d1ccdb82c6a7a9769bc84f55dd4ca6057
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sat Jul 3 19:52:52 2021 +0600

        fix(ammunition): fixes some missing ammo icons

        PR #5881

    commit 04a19d142708939d96fa057769df7a617acf7a49
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sat Jul 3 19:51:25 2021 +0600

        fix(build): fixes skulls dupe, some weird xenobehavior, reentering qdeleted bodies, etc

        PR #5891
        - Исправлена возможность бесконечно спавнить черепа и ошмётки с помощью голов.
        - Из голов дион и ксеноморфов больше нельзя доставать человеческие черепа.
        - Добавлена задержка при ударах по мехам.
        - Грудоломы теперь появляются через секунду после разрывания человека. Чтобы их не убивало летящими запчастями тел в тесных помещениях.
        - При использовании Re-Enter Corpse добавлена проверка на нахождение тела в нуллспейсе. Поскольку сейчас практически никакие мобы нормально не куделятся, попытка вернуться в такого моба приводила ко всяким паранормальным явлениям вроде появления в виде мозга/хэдкраба-зомби в космосе/на астероиде/в жопе админа.
        - Исправлена возможность появления у королевы ксеноморфов нейролейса.
        - Исправлено превращение дрона в грудолома вместо королевы при использовании Evolve, если дрон появился не "натуральным" путём, а с помощью кривого использования админских кнопок.
        - Чутка ослабил петлю на шее у GC, прибравшись в Destroy() у мобов. Большая часть мобов всё равно ещё где-то как-то референсится, но по крайней мере /mob/living/simple_animal уже нормально удаляются, что радует.
        - Теперь при получении роли ксеноморфа будут выводиться разные сообщения в зависимости от того, является ли моб грудоломом или ксеноморфом.

    commit a0cfa0b55893eebab83a9e14426d61c6309d9593
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Sat Jul 3 19:39:22 2021 +0600

        tweak(sfx): cuts a delay before the actual sound in smash.ogg

        PR #5883

    commit c56487aee7d78996b705e24ccb177985185ff044
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Fri Jul 2 15:29:53 2021 +0300

        tweak(R&D machinery): changes plain browser to TGUI

        PR #5859

    commit c3b39e701cc23d0e932f3ad55e0d6a406f33fbff
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Fri Jul 2 04:59:16 2021 +0600

        fix(stun gloves): fixes broken initialize

        PR #5865
        - Стан-перчатки инициализировались через жопу.
        - Заспавненные/намаппленные стан-перчатки теперь по умолчанию выбирают базой рабочие перчатки. До этого - получалось поломанное нечто.
        - Попытка обрезать/обмотать проводами перчатки-хамелеоны теперь выдаёт очевидное сообщение, как у боксёрских перчаток.

    commit e728e699437fe3281ea11b066031fa4cb3bdc8ce
    Author: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
    Date:   Fri Jul 2 00:49:48 2021 +0300

        fix(wizard): fixes knock and contract skills

        PR #5848
        - Заклинание "Knock" открывает обесточенные шлюзы.
        - Исправлены заклинания взаимодействия мага с учениками.
        - Заклинание "Return to Master" телепортирует ученика к магу, а не наоборот.

    commit b9ce904848d1594e3427595210448d025e04ecde
    Author: Artemonim <Artemonim@yandex.ru>
    Date:   Thu Jul 1 00:26:42 2021 +0300

        tweak(consoles): engineering consoles have nanoword preinstalled

        PR #5827

    commit f564a1a143ae87dad17fd26fd752f0840b3e4453
    Author: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
    Date:   Wed Jun 30 22:15:24 2021 +0300

        fix(mimics): defaults their obj type to crate

        PR #5841
        - Заспавненные мимики теперь по умолчанию будут ящиками, так что больше никаких бессмертных ебак.

    commit ba006fe87c6766d16560558c7dcb171cc4bdd81a
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Wed Jun 30 15:33:48 2021 +0300

        fix(vending.dm): Runtime in vending.dm, 188: Cannot execute null.GetIdCard().

        PR #5836

    commit 378d1e920f0ff82a728636e7b5d4d6b38bdee1cc
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Wed Jun 30 18:21:48 2021 +0600

        tweak(backpacks): renews medical, sci and hydro backpacks worn icons

        PR #5840
        -  Обновлены спрайты некоторых рюкзаков (у медиков, химиков, вирусологов и учёных) на спине. Теперь они выглядят соответственно спрайтам самих рюкзаков.

    commit b4f83b195e263be1898dcc7a6047de7d277f1572
    Author: Epicus <epicus7@gmail.com>
    Date:   Wed Jun 30 13:33:54 2021 +0300

        feat(docs): add "as anything()" section to contributing.dm

    commit dda65c87bbeed067784762688724d9dcbfc4b18d
    Author: Artemonim <Artemonim@yandex.ru>
    Date:   Wed Jun 30 09:38:58 2021 +0300

        fix(viruses): Runtime in helpers.dm, 92: Cannot read null.name

        PR #5822

    commit 0140fbacc12071825b1caa35981e372d909ac8c7
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Tue Jun 29 23:34:20 2021 +0300

        Changelog 29/06/21 (#5821)

    commit 4906d99f12c5e9b8e0b4155cfeba1ebee70761c5
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Wed Jun 30 02:29:48 2021 +0600

        fix(build): allows the build to compile

        PR #5820
        - Исправлена ошибка с инклюдами из одного из ПРов.

    commit 91428ffaf6121dd51bb4edce82bca9f634efd2cb
    Author: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
    Date:   Tue Jun 29 23:00:02 2021 +0300

        fix(wizard): teleporting from cryo no longer produces cringe

        PR #5750
        - Добавлена проверка на нахождении в криокапсуле, если маг кастует из нее, то его сначала оттуда выкинет, и после этого произойдет блинк/телепорт.

    commit da6bdfb71ee330cc6b6d84bd27f32b8889fc333d
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Wed Jun 30 01:58:16 2021 +0600

        tweak(events): reduces viruses' weight, makes some events non-oneshot

        PR #5818
        - Уменьшен вес всех трёх уровней вирусов: Space Cold Outbreak (20 -> 15 за медика), Viral Infection (40 -> 20 за медика), Plague Infection (35 -> 20 за медика). Plague Infection теперь одноразовый ивент.
        - Space Vines, Carp Migration, Carp School, Appendicitis, Lost Carp - больше не одноразовые ивенты.
        - Вес Appendicitis снижен (15 -> 3 за медика), ибо теперь может произойти больше одного раза за раунд.
        - Вес Wormholes увеличен (10 по умолчанию -> 10 по умолчанию + 1 за каждого игрока).

    commit 55de0cb982305f4ef51b80f64a1be65e8ba27077
    Author: No-Tips <75416333+No-Tips@users.noreply.github.com>
    Date:   Tue Jun 29 22:57:25 2021 +0300

        fix(corgi): allows corgies to wear hats... again

        PR #5698
        - Корги снова могут носить шапки.

    commit 15b4906a51d8d8dcc1c4941f6aae645b98d302cb
    Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
    Date:   Tue Jun 29 22:56:27 2021 +0300

        tweak(chat): adds graphic emoji support

        PR #5819

    commit 1c0ab5c2147b0575d7ac5d1eab009f3878234c65
    Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
    Date:   Wed Jun 30 01:52:27 2021 +0600

        fix(medicine): fixes carotid artery typo

        PR #5806

commit 03564fc83ae49c70b41839ff96342f1e050b8a6b
Author: Oubi <lito5443@gmail.com>
Date:   Sun Aug 1 17:26:04 2021 +0300

    tweak(trash): trashy dishes can now be stacked

    PR #5684
    - Добавлена возможность стакать тарелки и их разновидности.

commit 71c0ff53988ff0c204dc37f1da49ccebe991f959
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Tue Jul 27 21:12:29 2021 +0300

    tweak(contracts): empty contracts are no longer a thing

    PR #6233

commit c147bb62d14cf71c1625814333fe978cd7829643
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Mon Jul 26 19:01:43 2021 +0300

    tweak(contracts): admin menu now display closed contracts

commit e644add6cfe537f94155ecdae942e2a970e0a19b
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Mon Jul 26 20:42:57 2021 +0600

    tweak(hear_sleep): fixes message formatting

    PR #6221

commit 7144a006a554e12d7b3e21de6e6cddb3626f5e66
Author: SerJo <60974487+SerJo2@users.noreply.github.com>
Date:   Tue Jul 27 00:31:09 2021 +1000

    fix(name): Changed Happy Christmas to Merry Christmas

    * Changed Happy Cristmas to Marry Cristmas

commit c2b49505adfae3d3413d0855af82a90df512bba6
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Mon Jul 26 20:30:05 2021 +0600

    feat(tools): exclusive tools for CE

commit 5560066f742c9199485e9636a657fe8a1ba811a5
Author: Oubi <lito5443@gmail.com>
Date:   Mon Jul 26 17:29:11 2021 +0300

    fix(rig healthanalyzer): fix health analyzer display output

    * rig healthanalyzer fix

commit 5ebac998a4868b2a694e94a998d6f53fbaebd667
Author: quardbreak <44920739+quardbreak@users.noreply.github.com>
Date:   Mon Jul 26 13:38:13 2021 +0300

    add(toys): plush snail

commit cc5d72c5393c0179a91f028b37ffa003f6d16d6d
Author: Nod404 <87613255+Nod404@users.noreply.github.com>
Date:   Sun Jul 25 22:23:55 2021 +0300

    remove(news): old news deleted

commit f9af41439d6efd71cf72555a1a4d666df65563ba
Author: SerJo <60974487+SerJo2@users.noreply.github.com>
Date:   Mon Jul 26 05:20:29 2021 +1000

    tweak(disposal): now you can climb inside in handcuffs

commit 3affee399f4ad2c2ec47c4b622ddb12af269c68d
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Sun Jul 25 22:18:30 2021 +0300

    remove(unused scripts): unused scripts removed

commit e6b77f970d66e3ffe61c3126580ab458d6b0e51f
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Sun Jul 25 22:17:29 2021 +0300

    fix(global_vars): global_vars/Destroy() are working correctly

commit 2a18c356b28472056e502c67a3005d914cc0a1cd
Author: Oubi <lito5443@gmail.com>
Date:   Sun Jul 25 22:11:19 2021 +0300

    fix(changeling): transform wear drop fix

    * Transform fixes

commit 3a39a97ac0f0b93daab8afe6c97776da2ce541a9
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Sun Jul 25 18:20:00 2021 +0300

    tweak(viruses): renames virus subsystem so we don't look like a bunch of morons

    PR #6219

commit d231575fd4a8d4310338432abedaffc09c6cf397
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Sun Jul 25 01:45:36 2021 +0300

    fix(borg): borgs now have sprite for open panel

    PR #6199

commit c2333daadde77b5aac70a7cd22854bc494f75d69
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Sat Jul 24 00:36:33 2021 +0300

    optimization(RnD console): tiny optimization for a RnD console

commit 3c23d4288306921fb2fce387ea6a6ee80057c976
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Fri Jul 23 23:53:56 2021 +0300

    feat(medical machinery): assemble and disassemble medical equipment

    * feat(medical machinery) add construction for medical machinery

    * feat(optable) add optable deconstuction/construction

commit 7354df0c7c313c34b12152e188eff2480172898a
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Thu Jul 22 20:39:29 2021 +0600

    fix(crates): fixes multitool hacking

    PR #6183
    - Исправлена невозможность взлома ящиков мультитулом.

commit fc77daf267ab844ca34e25ab104fe2a94d8db347
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Thu Jul 22 15:20:47 2021 +0300

    tweak(TG UI): removes some tooltips

    PR #6181
    - ...cause they cause lag

commit 54870e3f8334108fb072b917cd28ecf24e2cf7bd
Author: malboec <86184187+malboec@users.noreply.github.com>
Date:   Thu Jul 22 01:42:26 2021 +0500

    fix(machete): machetes now use a proper compile-time icon state

    PR #6178

commit 17f671768395f75b8aa8763beb477af4a165fc5c
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Thu Jul 22 01:28:19 2021 +0600

    tweak(gas stand): updates icons

    PR #6161

commit 331c41dc49375f2168f53e1878ffb50ae2340aaf
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Thu Jul 22 01:28:00 2021 +0600

    tweak(syringes/screwdrivers): syringes and screwdrivers are displayed vertically while in storages

    PR #6163

commit fac69c3099b4deed5227d10154ad0647bb44eb69
Author: Mark <42303535+Zert0X@users.noreply.github.com>
Date:   Wed Jul 21 22:13:50 2021 +0300

    tweak(antags): adds vampire and thrall icons for antaghud

    PR #6168

commit 7b528fd87f0fbbe2696d669c985bf349d349d84d
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Wed Jul 21 21:35:07 2021 +0300

    fix(story log): fixes ckey leak

    PR #6170

commit 78cb6de884366956a258f91b20c284e49360a423
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Wed Jul 21 21:34:04 2021 +0300

    fix(pedal generator): fixes runtime undefined variable /mob/living/silicon/robot/drone/var/handcuffed

    PR #6171

commit e39e022d18496c8fe6751e9c4524d56addf3dde6
Author: Nod404 <87613255+Nod404@users.noreply.github.com>
Date:   Wed Jul 21 19:53:48 2021 +0300

    tweak(graffiti): removes yoba-face

    PR #6172

commit 0a33ab5dc418f9a93392e2b601d894d189f205ed
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Wed Jul 21 00:46:34 2021 +0300

    Changelog 20/07/21 (#6162)

commit f919e446eff0feebd44a5f28d0d377bef60db961
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Wed Jul 21 03:34:16 2021 +0600

    tweak(liver): adds safe toxic damage values

    PR #5039
    - Теперь у печени есть запас токс-урона, который она может сдержать. Если запас не превышен - токсины будут постепенно выводиться из организма, не нанося урона. Если превышен - как и раньше, органы будут получать урон, начиная с печени и почек.

commit 182195e4f9db569591837e044fd4d47fb20eab07
Author: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>
Date:   Tue Jul 20 23:56:15 2021 +0300

    tweak(robots): borgs can deflect projectiles with energy swords

    PR #6153

commit 7bf9b06b8bf6a9a226ecd4ca88bd4e4abc89c6aa
Author: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>
Date:   Tue Jul 20 22:56:49 2021 +0300

    fix(inflatables): borgs can no longer switching state remotely

    PR #6160

commit 4abb6b6910c42815b454e0f8966c4ec1c851750e
Author: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
Date:   Tue Jul 20 19:24:05 2021 +0300

    fix(nullrod): fixes nullrod inability to decultify properly

    * fix(nullrod): fixes nullrod inability to decultify properly
    * delete(de-cultify): removes unused code

commit 9d4adeaddfbb9c663eb235d1dd936c8be7cf66c6
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Tue Jul 20 19:22:01 2021 +0300

    fix(runtime): unknown sound: science

commit d71bdc73755ab1f02f4dfe539be3aabe0b0d5d45
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Tue Jul 20 16:12:10 2021 +0300

    tweak(rcd) fixes assorted bugs

    PR #6152

    * fix rcd

    * Update code/game/objects/items/weapons/RCD.dm

    Co-authored-by: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>

    Co-authored-by: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>

commit 297c45d724cde4e91f6ef3d53b8978512cd89db8
Author: hard-res3t <59255670+hard-res3t@users.noreply.github.com>
Date:   Tue Jul 20 11:25:37 2021 +0300

    tweak(map): fixes assorted bugs

    PR #5908
    - Починил доступ к генератору гравитации.
    - Перенес одну стенку чтобы лестница не уходила в стену в техах у отбытия.
    - Добавил воксам на базу азот.
    - Починил невидимые деревья в снежном лесу на голодеке.
    - Изменён спрайт голопеска.
    - В кабинет АВД добавлен шреддер и планшет.

commit b302f4b100e56c3d076bf1757699edf71ddd964c
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Tue Jul 20 14:18:46 2021 +0600

    feat(sledgehammer): adds true space asshole's weapon

    PR #5904

commit 536d36db5bc1a6585f60dcba72d3ab642f1e32cb
Author: hard-res3t <59255670+hard-res3t@users.noreply.github.com>
Date:   Tue Jul 20 09:34:24 2021 +0300

    tweak(Frontier): fixes assorted bugs

    PR #5844
    - У учёных есть доступ к своим техам.
    - В кабинет АВД добавлен шреддер.
    - В кабинет АВД добавлен планшет.
    - Убран доступ шахтеров к докам карго шаттла.
    - Добавлена кнопка для шторок в переговорную глав.
    - Добавлен азот на базу воксов.

commit c68b289646c1861e560779f90c39d501127bfdc1
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Tue Jul 20 09:22:31 2021 +0300

    feat(ai customs): adds example states for AI custom icons

    PR #5292

commit 04d6a8cd24d038adea822749419b4fb539249083
Author: BWJes <86688350+BWJes@users.noreply.github.com>
Date:   Tue Jul 20 12:53:45 2021 +0700

    tweak(janitor): adds janitorial belt to janitor's closet

    PR #5945

commit 0f2a99e79b61d0d84d88c92b970f3c9f9ac1aca1
Author: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
Date:   Tue Jul 20 08:49:52 2021 +0300

    fix(armblades): fixes description of armblade and EMP gibbing fake armblades

    PR #6016
    - Фейковый армблейд больше не гиббается от ЭМИ.
    - Исправлено описание армблейда, убрана строчка про рефлект пуль.

commit 9f6d5da2e9be836479ef339d7ac3d5d466158ad6
Author: Nod404 <87613255+Nod404@users.noreply.github.com>
Date:   Tue Jul 20 08:11:07 2021 +0300

    tweak(smokables): reduces light power and range

    PR #6137
    - Уменьшено свечение от горящих сигарет, сигар и т.д.

commit 63bcaf53c4e7374ba85c8f01e25cebd4517dc7cf
Author: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>
Date:   Mon Jul 19 22:02:04 2021 +0300

    tweak(changeling): less blood more difficulty

    * tweak(changeling): added steal identity objective, reduced the required amount of absorbed DNA
    * prevent transformation trick

commit 1b10f19181d3949f27250ce15b0d4f6d6df3da0a
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Mon Jul 19 21:36:25 2021 +0300

    fix(runtime): respawn character now works prop

    * add try-get wrapper for preferences

    * fix runtime in ignore

    * remove cringe fix

    * add proximity_checker nullable check

    * add nullable check to camera shake

    * fix roundend action runtime

    * fix bad icon operation runtime

    * fix respawn_character

commit b44bb796d6e130ae4e43763e136a015c6756be7c
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Tue Jul 20 00:35:06 2021 +0600

    fix(closet): can be repaired after hack

commit 92179bbbcc445343c2032796008607faaa4e0a16
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Mon Jul 19 21:32:26 2021 +0300

    tweak(portal): move all objects from portal to loc

    * tweak(portal) move all objects from portal loc

    * add wait timer

    * update checks

commit 93bda9787741daa25048e5e034c4561e37989c3a
Author: Mark <42303535+Zert0X@users.noreply.github.com>
Date:   Mon Jul 19 20:40:30 2021 +0300

    (tweaks): vampire skills rebalance

    * Vamp can see in darkness

    * Larger text in chat

    * Some abilities tweaks

    * power use cost to var && some spaces

commit 163c6d1a3c3abc87133784291c5bbc24658e58c8
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Mon Jul 19 19:16:37 2021 +0300

    tweak(vending machines): removes flying burgers

    PR #6129
    - Убраны летающие бургеры с фона UI вендоматов
    - :-(

commit 17bc0608134bfcef4f46448be2a6026f24576c62
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Mon Jul 19 22:15:23 2021 +0600

    tweak(snacks): removes cannabar, adds hemp2gen

    PR #6125

commit c1bcd0841e80c0d1f94eb29a9db2b427cb8ed448
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Mon Jul 19 20:26:47 2021 +0600

    tweak(projectiles): reduces stunshells' penetraction modifier

    PR #6107

commit dd9bc6d74dbb2a9a7243e58be3fb9b6753d7a397
Author: Nod404 <87613255+Nod404@users.noreply.github.com>
Date:   Mon Jul 19 15:36:53 2021 +0300

    fix(candies): adds "candy" state, changes default state

    PR #6121

commit 2869dda30f13935b5718f2494f683233b5655200
Author: Oubi <lito5443@gmail.com>
Date:   Mon Jul 19 09:40:17 2021 +0300

    tweak(eswords): swords emmit light when active

    PR #5993
    - Включенные энергетические мечи и кинжалы теперь светятся во включенном состоянии.

commit 5a6e490ac1f1808086f65acab167efbb411dee7a
Author: Oubi <lito5443@gmail.com>
Date:   Fri Jul 16 23:29:12 2021 +0300

    fix(viruses): fixes antibodies mixing

    PR #5953
    - Исправлено смешивание антител. Теперь их часть не пропадает.

commit 7517c965f6483ecfdf1612462e2215e13c57d8ab
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Sat Jul 17 02:24:37 2021 +0600

    feat(custom items): adds AmiClerick's custom labcoat

    PR #6103
    - Добавлена кастомка для AmiClerick.

commit 2f3eafe86930a9aa124cbc829d46980c135200eb
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Fri Jul 16 22:36:06 2021 +0600

    fix(antagonist): fixes xenomorphs and borers spawn

    PR #6100
    - Бореры больше не наследуются от ксеноморфов;
    - Исправлены возможные проблемы со спавном ксеноморфов;

commit 877701d44f0782f0912c08b997d1a4374cb015d9
Author: Epicus <epicus7@gmail.com>
Date:   Fri Jul 16 12:07:26 2021 +0300

    fix(antagonist): fix update_antag_mob

    PR #6097
    - Исправлен спавн воксов-рейдеров на станции вместо их собственной локации.

commit 06822da9d1db04f189098b22cefe47fe900da64b
Author: Mark <42303535+Zert0X@users.noreply.github.com>
Date:   Fri Jul 16 07:45:47 2021 +0300

    tweak(build): removes ban_everywhere option

    PR #5939

commit 3c624442240c84aa9e6a01ebafc70b88d42b83a9
Author: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Date:   Fri Jul 16 03:02:04 2021 +0600

    fix(prosthesis): fixes absent slim male chest prosthetic icons

    PR $6089

commit 04f10db5800eb32a18a279d9377c90a6acc8b3e4
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Thu Jul 15 23:26:13 2021 +0300

    fix(grab): adds more checks into processing proc

    PR #6088
    - Добавлено больше проверок в процессинг грабов;

commit c9e596da63f67951a19abb76f86fa494a9192c86
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Thu Jul 15 23:04:12 2021 +0300

    fix(changeling): fixes changeling reagent mutation

    PR #5989
    - При становлении генокрадом через генкореагент космонавтики больше не получают задания генокрадов.
    - Космонавтики, ставшие генокрадами через генкореагенты или способность Division теперь показываются в списке антагонистов в конце раунда.

commit b9a8cd4f1ec1d3e6dbedf0f745bd50c4023181d8
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Thu Jul 15 16:46:20 2021 +0300

    feat(logging): adds story and integrated circuits logging

commit e237ad328ca3b9205328e901fce9481e1539af38
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Wed Jul 14 11:56:01 2021 +0300

    purge(code): remove a bunch of unused code

commit 5fda78a42217dc9240af314954cbc69c75291ce1
Author: Max <47484306+MaxFreemen@users.noreply.github.com>
Date:   Wed Jul 14 11:51:29 2021 +0300

    tweak(virusdish): new sprites for "virus dish".

commit cef3fdb91fc242f420ba7fb1ce250dced4bf40af
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Mon Jul 12 20:17:01 2021 +0300

    Генерация чейнджлога на 12.07.2021 (#6049)

commit 728d573e5de5582652cf139ba6509dfb335336d0
Author: Doster <63081538+Doster-d@users.noreply.github.com>
Date:   Mon Jul 12 18:25:06 2021 +0300

    feat(subsystems): explosion and virus subsystem update

    PR #6008
    - портирование системы взрывов с ТГ;
    - виры теперь находятся в правильном приоритете.

commit 11472e400567d8011959860a99817d343a3f332a
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Mon Jul 12 17:57:47 2021 +0300

    Del(overmap): remove overmap

    PR #5921
    - Удалена система овермапов и всё что с ней связанно.

commit 283b57f42f76a88394da8a8071bc8d71660824fe
Author: igorsaux <55196698+igorsaux@users.noreply.github.com>
Date:   Mon Jul 12 13:56:13 2021 +0300

    fix(R&D machinery): makes rnd console be more optimized

commit 67b493043e3f7b93467e69810b96a1b464a9138f
Author: No-Tips <75416333+No-Tips@users.noreply.github.com>
Date:   Sun Jul 11 23:51:32 2021 +0300

    fix(metroid): Fixes adamantine's metroid sprite

Artemonim committed:
Revert "Squashed commit of the following:"

This reverts commit df9d644b4ca9a2a6ac43c05628fd3aa10b1d5974.

Artemonim committed:
Запрет на открытие дверей симплмобами

Artemonim committed:
прожигание шлюзов термитом

ghost commented:
Ничего не хочу сказать, но автору следовало бы разделить своё творчество на несколько кусков. Пихать различные фичи и багфиксы, хоть и относящиеся к одному и тому же объекту, - не есть круто. Это будет проблематично как и ревьювить, так и геймдизайнить. 

К тому же последний комментарий за 16 июля.
Закрываю.
